src/colors.config.ts

```diff
@@ -5,7 +5,7 @@ const colors = {
     bgOverlay80: 'rgba(0, 0, 0, 0.8)',
     bgOverlay50: 'rgba(0, 0, 0, 0.5)',
     bgOverlay30: 'rgba(0, 0, 0, 0.3)',
-    black: 'rgba(0, 0, 0, 1)',
+    black: 'rgba(0, 0, 0, 1)', // should be unused now
```

Comment:
From a quick search, this looks like the only place still using `'black'`:

https://github.com/cybersemics/em/blob/31c4ea737db3b453f38799759a5c26198e573a2f/src/components/ColorPicker.tsx#L95

Should this be updated?

Reply:
Yes, sorry, I'll remove that later, I added that afterwards, and forgot to remove the comment

Reply:
Thanks! Are you planning to make that change in this PR or a future one?

Reply:
Sorry about that, I've just changed it.